srml/council/src/seats.rs

```diff
@@ -487,15 +487,17 @@ decl_module! {
 		/// Set the desired member count; if lower than the current count, then seats will not be up
 		/// election when they expire. If more, then a new vote will be started if one is not
 		/// already in progress.
-		fn set_desired_seats(#[compact] count: u32) {
+		fn set_desired_seats(origin, #[compact] count: u32) {
```

Comment:
one can still write `set_desired_seats(root, #[compact] ...`
the generated function won't have the first argument then.

Then the test call line 2325 doesn't have to be changed.

But my comment is more about this mecanism of writing origin as root is not defined in the doc, is it ? if no it might have been introduced for priviledge function actually. is something to be thought here ?

Reply:
I have since removed this functionality as I believe it goes against the whole point of this PR.